### PR TITLE
Add app script to compute per-residue Shannon entropy

### PR DESCRIPTION
**Pull request by sukritsingh**
Was digging around my old fork and found an old app script I had written to quickly compute Shannon entropies. The underlying functionality is fairly simple for anybody familiar with entropy stuff, moreso just a "consistent convenience" thing I had lying around. 

Would you folks be interested in this PR? If so I can spruce up the doc strings a bit.

### CODE REVIEW

**Comment by lgsmith** [comment]
I think yeah why not. Sometimes having simple stuff like that lying around
is helpful for people trying to discern how something is coded up--in the
context of a more sophisticated tool it can be hard to follow.

**Comment by Justin-J-Miller** [comment]
Agreed!On May 28, 2024, at 9:59 PM, lgsmith ***@***.***> wrote:﻿
I think yeah why not. Sometimes having simple stuff like that lying around

is helpful for people trying to discern how something is coded up--in the

context of a more sophisticated tool it can be hard to follow.



On Tue, May 28, 2024 at 9:16 PM Sukrit Singh ***@***.***>

wrote:

**Comment by sukritsingh** [comment]
Alright - docstrings added for both the new shannon entropy script and for `compute_cards`! 

Admittedly not the largest contribution of code, but hopefully useful! Let me know if yall want any changes.

**Comment by sukritsingh** [comment]
Nothing to apologize for! I'm unsure about needing additional tests - the app script uses `enspara.info_theory.entropy` under the hood which we have a whole testing script for at [`enspara/test/test_entropy.py`](https://github.com/bowman-lab/enspara/blob/master/enspara/test/test_entropy.py). I think the only "new" thing the app script does is sum up the dihedral entropies to generate residue-level entropies, which is done in the script itself.

If you think more are warranted I'm happy to add some, but I figure since it's not modifying any of the core utilities it's not really necessary?
